test/test_special_elixirs.jl

```diff
@@ -345,8 +344,8 @@ end
     end
 
     @timed_testset "elixir_euler_ad.jl" begin
-        @test_trixi_include(joinpath(examples_dir(), "special_elixirs",
-                                     "elixir_euler_ad.jl"))
+        @test_nowarn_mod trixi_include(joinpath(examples_dir(), "special_elixirs",
+                                                "elixir_euler_ad.jl"))
```

Comment:
If I see that correctly, `@test_trixi_include` does nothing other than `@test_nowarn_mod trixi_include` plus a little `println` when not used with the kwargs `l2` and `linf`.

Reply:
Why do you change it then?

Reply:
Because it also sets `maxiters`, which doesn't exist and would make sense in this elixir.

Reply:
I see, thanks!